docs/site/translation.md

```diff
@@ -126,6 +133,14 @@ If you do not supply this, that combination will fall back to using the “stand
 in English that would produce “tomorrow, 3:00 PM”. If instead you want the same combining behavior for a relative date with a single time as for a
 fvfixed date with single time (as was the case in CLDR 47 and earlier), then for each length style copy the existing “atTime” form to the new “relative” form.
 
+#### Missing date & time patterns
+
+💡🆕 Currently dates and times are sometimes poorly formatted or ambiguous due to missing patterns. The additional patterns have been added to resolve this issue:
+
+  - GyM and GyMEd
+  - Time formats: EBh, Eh, EH when there is an existing EBhm, Ehm, and EHm in the calendar respectively.
+  - Changed HH patterns in available and internval formats to include a reference to hour since seeing an hour number alone is ambigious.
```

Comment:
```suggestion
  - HH patterns in available and internal formats have been changed to include a reference to hour since seeing an hour number alone is ambigious.
```